templates/projects/project.html

```diff
@@ -30,6 +29,7 @@
                     <td><p>Number of tasks in progress:</p></td>
                     <td><p>{{num_locked_tasks}}</p></td>
                 </tr>
+                {% if current_user.admin or current_user.subadmin or current_user.id in project.owners_ids %}
```

Comment:
Looks like this line of change was from RDISCROWD-5481, which we decided to hold on for the deployment. There is a concern for the performance as the current implementation of ```n_available_tasks``` is slow. If we  let every contributor of the project to see this data, it could negatively affect our performance.

Reply:
I believe we had changed the computation for `n_available_tasks` to correct this (return 0)? The above change is displaying the HTML fields, which have already been computed on the server-side. Would a performance impact still be present if this change is only showing the previously hidden field, but computation is performed regardless via server-side?

Reply:
We changed ```n_available_tasks``` for "category reserved" projects only and it returns a fixed number 10 at this moment.  Different users will get a different ```n_available_tasks```. The interesting question is that, if the Jinja2 template does not display a field (like our case wrapped with IF condition), will it still compute a field on the server-side beforehand? I tried to look into the official [document](https://jinja.palletsprojects.com/en/2.11.x/templates/#list-of-control-structures), and it says "The if statement in Jinja is comparable with the Python if statement. In the simplest form, you can use it to test if a variable is defined, not empty and not false:". So, still not very clear it will compute the data beforehand. But I believe Jinja2 should not do that. Adding @dchhabda for comment.

Reply:
@kbecker42 , looks like your PR branch needs rebase as its lagging latest main branch changes. Check this [here](https://github.com/bloomberg/pybossa-default-theme/blob/17efa568c69b2f9a3214777b2bb530fe9b379588/templates/projects/project.html#L20). With rebase, of this PR branch, the issue would resolve.
Regarding Xi's comment, agree that as `n_available_tasks` has performance it, it makes sense to me to not expose it to all users until its optimized by cache changes that Xi is working on. Thanks!

Reply:
Removed change via https://github.com/bloomberg/pybossa-default-theme/pull/381/commits/71ebeb27bdb12648b41b6b4236de35fb54155379